packages/react-slider/src/components/Slider/stories/SliderControlled.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { Label } from '@fluentui/react-label';
+import { Slider, SliderProps } from '../../../index';
+
+export const Controlled = () => {
+  const [sliderValue, setSliderValue] = React.useState(160);
+  const onSliderChange: SliderProps['onChange'] = (_, data) => setSliderValue(data.value);
+
+  return (
+    <>
+      <Label htmlFor="control-slider">Control Slider [ Current Value: {sliderValue} ]</Label>
```

Comment:
Can you use `useId()` to generate these IDs? That prevents duplicate IDs on the page if the example is copy-pasted, and demonstrates the pattern people should be using in real components.